src/Akka.Streams.Kafka/Stages/Consumers/Abstract/SingleSourceStageLogic.cs

```diff
@@ -22,67 +22,53 @@ internal class SingleSourceStageLogic<K, V, TMessage> : BaseSingleSourceLogic<K,
     {
         private readonly SourceShape<TMessage> _shape;
         private readonly ConsumerSettings<K, V> _settings;
-        private readonly ISubscription _subscription;
 
         public SingleSourceStageLogic(SourceShape<TMessage> shape, ConsumerSettings<K, V> settings, 
                                       ISubscription subscription, Attributes attributes, 
                                       Func<BaseSingleSourceLogic<K, V, TMessage>, IMessageBuilder<K, V, TMessage>> messageBuilderFactory) 
-            : base(shape, attributes, messageBuilderFactory, settings.AutoCreateTopicsEnabled)
+            : base(shape, attributes, messageBuilderFactory, settings.AutoCreateTopicsEnabled, subscription)
         {
             _shape = shape;
             _settings = settings;
-            _subscription = subscription;
         }
 
-        /// <inheritdoc />
-        protected override void ConfigureSubscription()
+        protected override object LogSource
```

Comment:
Customized the `LogSource` to make it more obvious who is doing the logging here.